app/assets/javascripts/sephora_style_guide/video.js

```diff
@@ -29,35 +18,23 @@ function onYouTubeIframeAPIReady() {
 
 function onPlayerReady() {
   // bind events
-  $(document).on('click', '.close', function() {
-    var currentVideoId = $(this).closest('.modal-header').siblings().find('iframe').attr('id');
+  $(document).on('click', '.video-link', function() {
```

Comment:
Is `video-link` used for CSS? Then would be best to add one more `js-video-link`.